### PR TITLE
Mexc fetchFundingHistory

### PR DESCRIPTION
**Pull request by Dan-krm**
Added fetchFundingHistory to Mexc:
```
mexc.fetchFundingHistory (BTC/USDT:USDT)
491 ms
       symbol | code |     timestamp |                 datetime |      id |        amount
-----------------------------------------------------------------------------------------
BTC/USDT:USDT |      | 1643299200000 | 2022-01-27T16:00:00.000Z | 7423910 | 0.00076180624
BTC/USDT:USDT |      | 1643270400000 | 2022-01-27T08:00:00.000Z | 7416473 |  0.0014748588
2 objects
```

### CODE REVIEW

File: js/mexc.js [hunk 1]
```diff
@@ -2163,6 +2164,84 @@ module.exports = class mexc extends Exchange {
...
+            pageNum = 1;
+        } else {
+            pageNum = params['page_num'];
+        }
+        if (params['page_size'] === undefined) {
+            pageSize = 20;
+        } else {
+            pageSize = params['page_size'];
+        }
+        const request = {
```
Comment (kroitor) [inline js/mexc.js:2184]: We should not do lines 2172-2183, cause the request is extended with params-overrides in `this.extend` right before sending it out to the exchange, so whatever overrides the user has specified in `params` – those get into the request and override the defaults. So, we should just fill the `request` with the defaults. Also, we should not send a parameter, if it is optional on the exchange side.

Comment (Dan-krm) [reply js/mexc.js:2169]: Let me know if these changes are okay or not, it works with page_num and page_size commented out but the exchange api docs say that these fields are mandatory. I also removed the ArgumentsRequired for a symbol argument.

Comment (kroitor) [reply js/mexc.js:2169]: Most of the changes are ok, but if the user has supplied a `symbol` argument and the exchange supports it – then we must send the market['id'] to the exchange in the request. We should not send the market['id'] if the user has not supplied a symbol argument.

Comment (Dan-krm) [reply js/mexc.js:2170]: Sorry I missed that, I think it's good now

Comment (kroitor) [reply js/mexc.js:2170]: Thanks!

**Pull request merged**